shared/utils/yaml-to-shorthand.py

```diff
@@ -494,7 +494,7 @@ def from_yaml(yaml_file, product_yaml=None):
         rule.severity = required_yaml_key(yaml_contents, "severity")
         del yaml_contents["severity"]
         rule.references = yaml_contents.pop("references", [])
-        rule.identifiers = yaml_contents.pop("identifiers", [])
```

Comment:
it's not clear to me why this change was needed, it seems like defaulting to [] and having the assurance that rule.identifiers is always a list() would be OK and would work with the rest of the code

Reply:
discussed this with @cipherboy , turns out it needs to be a dict()